rundeckapp/grails-app/views/menu/_sysConfigNavMenu.gsp

```diff
@@ -63,5 +63,12 @@
       <g:message code="gui.menu.PasswordUtility"/>
     </g:link>
   </li>
+   <g:if test="${authRead}">
```

Comment:
this auth check does not seem correct, if the summary action requires user:admin access.